quick_actions.py

```diff
@@ -13,7 +13,7 @@
     LintError = persist.LintError
 
 
-class sl_quick_actions(sublime_plugin.TextCommand):
+class SublimeLinterQuickActions(sublime_plugin.TextCommand):
```

Comment:
We use lowercased `_` names now because searchability. Looks like I did not convert all command names.

Reply:
Of course, `sl_` was a mistake.

Reply:
Oh, lowercased is also good, I always thought the use of snake case in one context and camel case in another over complicated things.